config/dataworks_schema.yml

```diff
@@ -300,3 +303,18 @@ components:
       required:
         - title
       additionalProperties: false
+    Identifier:
+      type: object
+      properties:
+        identifier:
+          type: string
+        identifier_type:
+          type: string
+          enum:
+            - DOI
+            - redivis_reference
```

Comment:
To be consistent with the other enums in the schema, recommend "RedivisReference" and "SearchworksReference".